components/playlist/resources/components/playlistSelect.tsx

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import * as PlaylistMojo from 'gen/brave/components/playlist/mojom/playlist.mojom.m.js'
+
+interface Props {
+  playlists: PlaylistMojo.Playlist[]
+  selectedPlaylist: PlaylistMojo.Playlist | undefined
+
+  selectPlaylist: (playlistId: string) => void
```

Comment:
Optional nit: As this handles an event, I think the convention would be to name it with an `on` prefix

i.e.

`onSelectPlaylist`

but I'm kind of on the fence about it, because it's a command in this case. Up to you :D